features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -153,22 +153,45 @@ const CollapseMenuItem = styled(MenuItemButton)<{ isCollapsed: boolean }>`
   }
 `;
 
+const useIsDesktop = (): boolean => {
+  const [isDesktop, setDesktop] = useState<boolean>(true);
+
+  const updateMedia = useCallback(() => {
+    setDesktop(window.innerWidth > 1023);
```

Comment:
You probably know it already but this is basically duplicating the desktop breakpoint constant in the `theme.ts` file. I wonder if there's a way to create a utility function inside that file to get breakpoints in `px` unit (currently it's `em`).

Not sure if that's overkill here tbh. But constants like the `1023` can quickly become annoying in large codebases

Reply:
Now I remember, it's called a [magic number](https://refactoring.guru/replace-magic-number-with-symbolic-constant). To be honest it's kind of clear what it means here because the hook is named properly. Still it's considered a "code smell"

Reply:
Yep! I tried to use the breakpoint utility function before I realized that it's in `em`s. So I just googled what the pixel equivalent of 64ems(more or less) and ran with it 😅 . In hindsight, I could have at least put the value in a `const`. Thanks for the link!